app.js

```diff
@@ -14,38 +14,44 @@
 
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const querystring = require('querystring');
-const bcdBrowsers = require('@mdn/browser-compat-data').browsers;
-
-const express = require('express');
-const cookieParser = require('cookie-parser');
-const https = require('https');
-const http = require('http');
-const uniqueString = require('unique-string');
-const expressLayouts = require('express-ejs-layouts');
-
-const exporter = require('./exporter');
-const logger = require('./logger');
-const {parseResults} = require('./results');
-const storage = require('./storage').getStorage();
-const {parseUA} = require('./ua-parser');
+import fs from 'fs-extra';
+import querystring from 'querystring';
+import bcd from '@mdn/browser-compat-data';
+const bcdBrowsers = bcd.browsers;
+import esMain from 'es-main';
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import https from 'https';
+import http from 'http';
+import uniqueString from 'unique-string';
+import expressLayouts from 'express-ejs-layouts';
+import yargs from 'yargs';
+import {hideBin} from 'yargs/helpers';
+
+import * as exporter from './exporter.js';
+import logger from './logger.js';
+import parseResults from './results.js';
+import {getStorage} from './storage.js';
+import {parseUA} from './ua-parser.js';
+import Tests from './tests.js';
+
+const storage = getStorage();
 
 const appVersion =
   process.env.GAE_VERSION === 'production' ?
-    require('./package.json').version :
+    (await fs.readJson('./package.json'))?.version :
```

Comment:
Not sure why `?.` was added, the only way `await fs.readJson('./package.json')` could be null is if the file contains just the string "null", which is a surprising error we shouldn't mask. Suggest reverting:

```suggestion
    (await fs.readJson('./package.json')).version :
```